HeartbeatLogging/Sources/HeartbeatController.swift

```diff
@@ -108,12 +105,41 @@ public final class HeartbeatController {
     }
 
     // Synchronously gets and returns the stored heartbeats, resetting storage
-    // using the given transform. If the operation threw an error, assume no
-    // heartbeats were retrieved or set.
+    // using the given transform. If the operation throws, assume no
+    // heartbeat(s) were retrieved or set.
     if let heartbeatsBundle = try? storage.getAndSet(using: resetTransform) {
       return heartbeatsBundle.makeHeartbeatsPayload()
     } else {
       return HeartbeatsPayload.emptyPayload
     }
   }
+
+  /// Synchronously flushes the heartbeat for today.
+  ///
+  /// If no heartbeat was logged today, the returned payload is empty.
+  ///
+  /// - Note: This API is thread-safe.
+  /// - Returns: A heartbeats payload for the flushed heartbeat.
+  @discardableResult
+  public func flushHeartbeatFromToday() -> HeartbeatsPayload {
```

Comment:
reviewers: The existing v1 ObjC (`+ FIRHeartbeatInfo heartbeatCodeForTag`) API's implementation will instead call into this API. I will post ObjC wrapper in next PR.